File: Notification-Example 02/1) Hafeez/MainActivity.java
package com.codehafeez.notification;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import com.application.isradeleon.notify.Notify;
public class MainActivity extends AppCompatActivity {

    // read me => https://github.com/isradeleon/Notify-Android

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void function1(View view){
        Notify.build(getApplicationContext())
            .setTitle("Abdul Hafeez - Title")
            .setContent("Test Message Hello World")
            .setLargeIcon(R.drawable.image1)
            .largeCircularIcon()
            .setPicture(R.drawable.image2)
            .show();
    }
}

File: flavoursEx1/app/src/ChocolateFlavor/java/MainActivity.kt
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.codehafeez.flavoursex1.R
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setTitle(R.string.app_title)
    }
}